app/src/main/java/com/sdp/movemeet/Backend/FirebaseInteraction.java

```diff
@@ -33,7 +33,9 @@
         documentReference.addSnapshotListener(activity, new EventListener<DocumentSnapshot>() {
             @Override
             public void onEvent(@Nullable DocumentSnapshot documentSnapshot, @Nullable FirebaseFirestoreException e) {
-                if (textViewArray.length > 3) {
```

Comment:
So we don't have a case where the length of textViewArray might be smaller than "3" anymore? Because otherwise, we might want to handle this case as well... (like previously in the previous version of the MainActivity)

Reply:
The tests did not produce any errors but it is true that these edge cases should be tested and managed. But I'm not very good at this part, if someone is more clear, it might be better